docs/contrib/PlatformBringUp.md

```diff
@@ -334,112 +309,21 @@ make_tracks metal5 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
 Refer to the tech file to determine the x and y pitch and the offset.
 
 
-## PDN Configuration <a name="PDNConfiguration"></a>
+## PDN Configuration
 
-PDN is a utility that simplifies adding a power grid into the floorplan. With specifications given in the PDN configuration file, like which layer to use, stripe width and spacing, the utility can generate the metal straps used for the power grid. To create and configure a power grid, refer to [OpenROAD documentation](https://openroad.readthedocs.io/_/downloads/en/latest/pdf/) on pdngen. Following is a generic PDN configuration file:
+PDN is a utility that simplifies adding a power grid into the floorplan. With specifications given in the PDN configuration file, like which layer to use, stripe width and spacing, the utility can generate the metal straps used for the power grid. To create and configure a power grid, refer to [OpenROAD documentation](https://github.com/The-OpenROAD-Project/OpenROAD/blob/f864efbae4ffd4cc533f0e22c57cc7ab89618c44/src/pdn/doc/PDN.md) on pdngen.
```

Comment:
Is there a way to make these relative links? Ie. ../../../src/pdn/doc/PDN.md

Reply:
If that works in Markdown then I think that'd be the better choice. I tried adding a relative link in GettingStarted.md to PlatformBringUp.md but apparently the syntax was invalid (`../contrib/PlatformBringUp`).

Reply:
@rjridle Links should be relative to the docs folder, thus `./contrib/PlatformBringUp.md` should work. For links outside the docs folder, e.g., PDN above you need to use `./main/src/pdn/doc/PDN.md`.

Reply:
Made link relative.